### PR TITLE
Slides 

### PR DESCRIPTION
**Pull request by redfast00**
See  #187

* [X] cores vs nodes vs ... + sequential vs parallel software (multi-threaded vs distributed)
* [ ] example of job script for sequential & parallel job (+ explanation of what they do exactly)
* [X] overview of typical workflow
* [x] requesting software installations: use the form at https://www.ugent.be/hpc/en/support/software-installation-request (obviously specific to HPC-UGent)
* [X] Also need to check how much is explained about job scheduling, see slide 38 in slides dated `20180328`...

### CODE REVIEW

**Comment by redfast00** [comment]
I'm not entirely sure where to put the job scripts. I was thinking chapter 4, but there we haven't talked about parallel jobs yet.

**Comment by boegel** [comment]
@redfast00 How about creating a separate chapter for example job scripts (before Troubleshooting/FAQ)? Or maybe this overlaps too much with the examples that are already shown? Not sure if those are equivalent to the examples I use...

File: intro-HPC/ch_FAQ.tex [hunk 1]
```diff
@@ -56,3 +56,17 @@ \section{Can I use multple different SSH key pairs to connect ot my VSC account?
 
 Yes, and this is recommendend when working from different computers. Please see
 \autoref{sec:adding-multiple-keys} on how to do this.
+
+\section{I want to use software that isn't installed on the clusters}
```
Comment (boegel) [inline intro-HPC/ch_FAQ.tex]: `isn't installed` -> `is not available yet`

File: intro-HPC/ch_FAQ.tex [hunk 2]
```diff
@@ -56,3 +56,17 @@ \section{Can I use multple different SSH key pairs to connect ot my VSC account?
...
+
+\ifgent
+Please fill out the details about the software and why you need it in this form:
+\url{https://www.ugent.be/hpc/en/support/software-installation-request}.
```
Comment (boegel) [inline intro-HPC/ch_FAQ.tex]: When submitting the form, a mail will be sent to `\hpcinfo` containing all the provided information. The HPC team will look into your request as soon as possible you and contact you when the installation is done or if further information is required.

File: intro-HPC/ch_introduction.tex [hunk 3]
```diff
@@ -239,9 +239,22 @@ \subsection{Batch or interactive mode?}
 of your data without having to copy your data to your desktop and back.
 %TODO: only visualization nodes should be used interactively
 
+\subsection{What are cores, processors and nodes?}
+
+In this manual, the terms core, processor and node will be frequently used,
+so it's useful to understand what they are.
+
+Modern servers, also referred to as (worker) nodes in the context of HPC,
```
Comment (boegel) [inline intro-HPC/ch_introduction.tex]: drop space before `nodes`, and use `\emph{...}` to highlight they terms (nodes, processors, cores).

File: intro-HPC/ch_introduction.tex [hunk 4]
```diff
@@ -351,6 +377,25 @@ \subsection{What operating systems can I use?}
...
+    \item Transfer your files to the cluster (see \autoref{sec:filetransfer})
+    \item Optional: compile your code and test it (for compiling, see \autoref{ch:compiling-and-testing-your-software-on-the-hpc})
+    \item Create a job script and submit your job (see \autoref{ch:running-batch-jobs})
+    \item Get some coffee and be patient:
+    \begin{enumerate}
+        \item Your job gets into the queue
+        \item Your job gets executed
+        \item Your job finishes
+    \end{enumerate}
+    \item Move your results from the cluster to your computer.
```
Comment (boegel) [inline intro-HPC/ch_introduction.tex]: Study the results generated by your jobs, either on the cluster or after downloading them locally.

**Pull request merged**